03_inheritance/03_03_freeform_inheritance.py

```diff
@@ -12,3 +13,55 @@
 #
 # - A `Vehicle()` parent class, with `Truck()` and `Motorcycle()` child classes.
 # - A `Restaurant()` parent class, with `Gourmet()` and `FastFood()` child classes.
+
+class Restaurant:
+    """Restaurants"""
+    def __init__(self, name, region, to_go, price, meal) -> None:
+        self.name = name
+        self.region = region # Part of the world where food is from
+        self.to_go = to_go # Y or N
+        self.price = price # 1-$, 2-$$, 3-$$$
+        self.meal = meal # breakfst, lunch or dinner
```

Comment:
**nitpick**: Maybe it's better to call the `meal` property a plural `meals` because that communicates that it's a list better than the singular. Even if you just have one `meal` like `breakfast`, the way you'd need to use this is with a list `["breakfast"]`.